spec/factories/ethnicity.rb

```diff
@@ -13,5 +13,12 @@
       title { 'Asian or Asian British (Indian)' }
       description { 'A1 - Asian or Asian British (Indian)' }
     end
+
+    trait :b2 do
```

Comment:
This name isn't particularly revealing. How do you feel about `trait :black_british_african`?

Reply:
actually, it turns out the additional trait is not needed, I'll remove it